src/pages/Settings/TwoStepVerification.vue

```diff
@@ -1,38 +1,73 @@
 <template>
-	<div class="row">
-		<div class="column small-12 large-8">
-			<kv-settings-card title="2-Step verification">
-				<template v-slot:icon>
-					<!-- TODO: THIS ICON IS A PLACEHOLDER
+	<kv-settings-card title="2-Step verification">
+		<template v-slot:icon>
+			<!-- TODO: THIS ICON IS A PLACEHOLDER
 					Get correct icon assest from design, or remove this KvIcon -->
-					<kv-icon
-						class="icon"
-						title="Auto-lending On"
-						name="auto-icon-on"
-					/>
-				</template>
-				<template v-slot:content>
-					<p>Status:</p>
-					<p>Lorem Ipsum is simply dummy text of the printing and typesetting industry.</p>
-				</template>
-			</kv-settings-card>
-		</div>
-	</div>
+			<kv-icon
+				class="icon"
+				title="Auto-lending On"
+				name="auto-icon-on"
+			/>
+		</template>
+		<template v-slot:content>
+			<div>Status: <span v-html="MFAStatus"></span></div>
```

Comment:
This works, but I would prefer to not use v-html. A span with a conditional class and the styles in css I think would be better, the content can be a computed